locales/templates/app.pot

```diff
@@ -1202,3 +1251,159 @@ msgctxt "custom_tab_copy_url_action"
+#, c-format
+msgctxt "enable_search_suggestion_subtitle"
+msgid ""
+"To get search suggestions, %1$s needs to send what you type in the "
+"address bar to the search engine you've chosen. %2$s."
+msgstr ""
+
+#. Learn more string to be placed inside: enable_search_suggestion_subtitle
+msgctxt "enable_search_suggestion_subtitle_learnmore"
+msgid "Learn more"
+msgstr ""
+
+#. Label used for the no button for the prompt to enable search suggestions
+msgctxt "enable_search_suggestions_no"
+msgid "No"
+msgstr ""
+
+#. Label used for the yes button for the prompt to enable search suggestions
+msgctxt "enable_search_suggestions_yes"
+msgid "Yes"
+msgstr ""
+
+#. Message displayed when using a search engine that doesn't provide
+#. suggestions
+msgctxt "no_suggestions_message"
+msgid "Some search engines cannot show suggestions."
+msgstr ""
+
+#. Button to dismiss the message displayed  when using a search engine that
+#. doesn't provide suggestions
+msgctxt "dismiss_no_suggestions_prompt_button"
+msgid "Dismiss"
+msgstr ""
+
+#. Tips displayed on the home view
+#. Tip displayed on home view explaining how to disable tracking protection
+#. The newline here must be kept as the second half of the string is clickable
+#. for the user.
+msgctxt "tip_disable_tracking_protection"
+msgid ""
+"Site behaving unexpectedly?\n"
+" Try turning off Tracking Protection"
+msgstr ""
+
+#. Tip displayed on home view explaining how to add sites to homescreen
+#. The newline here must be kept as the second half of the string is clickable
+#. for the user.
+msgctxt "tip_add_to_homescreen"
+msgid ""
+"Get one-tap access to sites you use most\n"
+" Menu &gt; Add to Home screen"
+msgstr ""
+
+#. Tip displayed on home view explaining how to make Focus the default browser
+#. Argument 1 is the app name (Firefox Focus/Firefox Klar/etc).     The newline
+#. here must be kept as the second half of the string is clickable for the
+#. user.
+#, c-format
+msgctxt "tip_set_default_browser"
+msgid ""
+"Open every link in %1$s\n"
+" Set %1$s as default browser"
+msgstr ""
+
+#. Tip displayed on home view explaining how to add a custom autocomplete URL
+#. The newline here must be kept as the second half of the string is clickable
+#. for the user.
+msgctxt "tip_autocomplete_url"
+msgid ""
+"Autocomplete URLs for sites you use most\n"
+" Long-press any URL in the address bar"
+msgstr ""
+
+#. Tip displayed on home view explaining how to add a custom autocomplete URL
+#. The newline here must be kept as the second half of the string is clickable
+#. for the user.
+msgctxt "tip_open_in_new_tab"
+msgid ""
+"Open a link in a new tab\n"
+" Long-press any link on a page"
+msgstr ""
+
+#. Tip displayed on home view explaining how to request the desktop version of
+#. a website.      The newline here must be kept as the second half of the
+#. string is clickable for the user.
+msgctxt "tip_request_desktop"
+msgid ""
+"Rather see the full desktop site?\n"
+" Menu &gt; Request desktop site"
```

Comment:
Again not sure if the entity should be replaced here by the actual html tag. I can check about that